lib/fathead/python/redirect.py

```diff
@@ -225,6 +229,9 @@ def generate_redirects(f):
             if line.endswith('\\n'):
                 line = line[:-2]
                 line += '\t\t\t'
+            line = line.encode('unicode_escape')
```

Comment:
@souravbadami I see that the new approach is UTF-8 encoding several characters in some lines for example replacing apostrophes (‘) with \u2019 and non-breaking spaces with \xa0

These are then not being escaped so we're running into the same issue again.